enterprise/app/services/enterprise/billing/handle_stripe_event_service.rb

```diff
@@ -61,7 +61,7 @@ def ensure_event_context(event)
   end
 
   def features_to_update
-    %w[help_center campaigns team_management channel_twitter channel_facebook channel_email]
+    %w[help_center campaigns team_management sla channel_twitter channel_facebook channel_email]
```

Comment:
@pranavrajs @sojan-official @vishnu-narayanan seems like we enable all the features for any paid plan, even if it's available on the higher tier, should I fix it?

Reply:
Also need to verify the cases for the community edition and enterprise edition. 

- community edition:  we shouldn't show this page on the community edition ( or show ? )
- enterprise edition
   - paid with chatwoot meta disabled : should not show
   - non paid : show